drivers/storage/scaleio/tests/test-run.sh

```diff
@@ -0,0 +1,107 @@
+#SCALEIO_NODE3=$(aws cloudformation describe-stack-resources --stack-name ${CF_STACK_NAME} | jq '.StackResources[] | select(.LogicalResourceId=="ScaleIONode3")' | jq -r .PhysicalResourceId)
+#echo ${SCALEIO_NODE1}
+#echo ${SCALEIO_NODE2}
+#echo ${SCALEIO_NODE3}
+
+SCALEIO_NODE1_FQDN=$(aws ec2 describe-instances --instance-ids ${SCALEIO_NODE1} | jq -r '.Reservations[0].Instances[0].PublicDnsName')
+#SCALEIO_NODE2_FQDN=$(aws ec2 describe-instances --instance-ids ${SCALEIO_NODE2} | jq -r '.Reservations[0].Instances[0].PublicDnsName')
+#SCALEIO_NODE3_FQDN=$(aws ec2 describe-instances --instance-ids ${SCALEIO_NODE3} | jq -r '.Reservations[0].Instances[0].PublicDnsName')
+#echo ${SCALEIO_NODE1_FQDN}
+#echo ${SCALEIO_NODE2_FQDN}
+#echo ${SCALEIO_NODE3_FQDN}
+
+ssh-keyscan -H ${SCALEIO_NODE1_FQDN} >> ~/.ssh/known_hosts
+#ssh-keyscan -H ${SCALEIO_NODE2_FQDN} >> ~/.ssh/known_hosts
+#ssh-keyscan -H ${SCALEIO_NODE3_FQDN} >> ~/.ssh/known_hosts
+
+# Copy REX-Ray build to EC2 instance
+ssh $CF_EC2_USER@$SCALEIO_NODE1_FQDN "sudo mkdir -p /usr/bin"
+ssh $CF_EC2_USER@$SCALEIO_NODE1_FQDN "sudo mkdir -p /etc/rexray"
+scp ${TEST_BINARY} $CF_EC2_USER@$SCALEIO_NODE1_FQDN:/tmp
+scp ./config.yml $CF_EC2_USER@$SCALEIO_NODE1_FQDN:/tmp
+scp ./tests.sh $CF_EC2_USER@$SCALEIO_NODE1_FQDN:/tmp
+ssh $CF_EC2_USER@$SCALEIO_NODE1_FQDN "sudo cp -f /tmp/rexray /usr/bin"
+ssh $CF_EC2_USER@$SCALEIO_NODE1_FQDN "sudo cp -f /tmp/config.yml /etc/rexray"
+ssh $CF_EC2_USER@$SCALEIO_NODE1_FQDN "sudo chmod +x /tmp/tests.sh"
+ssh $CF_EC2_USER@$SCALEIO_NODE1_FQDN "sudo /usr/bin/rexray service restart"
+
+#ssh $CF_EC2_USER@$SCALEIO_NODE2_FQDN "sudo mkdir -p /usr/bin"
+#ssh $CF_EC2_USER@$SCALEIO_NODE2_FQDN "sudo mkdir -p /etc/rexray"
+#scp ${TEST_BINARY} $CF_EC2_USER@$SCALEIO_NODE2_FQDN:/tmp
+#scp ./config.yml $CF_EC2_USER@$SCALEIO_NODE2_FQDN:/tmp
+#scp ./tests.sh $CF_EC2_USER@$SCALEIO_NODE2_FQDN:/tmp
+#ssh $CF_EC2_USER@$SCALEIO_NODE2_FQDN "sudo /tmp/rexray /usr/bin"
+#ssh $CF_EC2_USER@$SCALEIO_NODE2_FQDN "sudo /tmp/config.yml /etc/rexray"
+#ssh $CF_EC2_USER@$SCALEIO_NODE2_FQDN "sudo chmod +x /tmp/tests.sh"
+#ssh $CF_EC2_USER@$SCALEIO_NODE2_FQDN "sudo /usr/bin/rexray service restart"
+
+#ssh $CF_EC2_USER@$SCALEIO_NODE3_FQDN "sudo mkdir -p /usr/bin"
+#ssh $CF_EC2_USER@$SCALEIO_NODE3_FQDN "sudo mkdir -p /etc/rexray"
+#scp ${TEST_BINARY} $CF_EC2_USER@$SCALEIO_NODE3_FQDN:/tmp
+#scp ./config.yml $CF_EC2_USER@$SCALEIO_NODE3_FQDN:/tmp
+#scp ./tests.sh $CF_EC2_USER@$SCALEIO_NODE3_FQDN:/tmp
+#ssh $CF_EC2_USER@$SCALEIO_NODE3_FQDN "sudo /tmp/rexray /usr/bin"
+#ssh $CF_EC2_USER@$SCALEIO_NODE3_FQDN "sudo /tmp/config.yml /etc/rexray"
+#ssh $CF_EC2_USER@$SCALEIO_NODE3_FQDN "sudo chmod +x /tmp/tests.sh"
+#ssh $CF_EC2_USER@$SCALEIO_NODE3_FQDN "sudo /usr/bin/rexray service restart"
+
+# Run tests
+ssh $CF_EC2_USER@$SCALEIO_NODE1_FQDN "sudo /tmp/tests.sh"
```

Comment:
Hi @dvonthenen,

The `tests.sh` file is not the test binary. See my above [comment](https://github.com/codedellemc/libstorage/pull/404/files#r99699916). Additionally, `output.txt` is not the coverage results and does not help to provide coverage. Please read the blog on [Go Cover](https://blog.golang.org/cover) for information regarding Go test coverage. We use [Codecov.io](https://codecov.io/gh/codedellemc/libstorage) for coverage, and the Go test binaries automatically produce coverage files that are consumed and reported to the Codecov site.

You can see the VFS test driver's test execution in the latest libStorage build from lines [743-746](https://travis-ci.org/codedellemc/libstorage/jobs/198507262#L743-L746). The lines [750-786](https://travis-ci.org/codedellemc/libstorage/jobs/198507262#L750-L786) show how we merge all of our produced test and coverage results and report them to Codecov.io.

Please update this PR to reflect the use of the ScaleIO test binary so that it is the file copied, executed, and whose results are copied back to Travis for coverage consumption.

Thanks!